### ISSUE TITLE
Gallery Block: Focus is lost after inserting images

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#47326]
### Description

In the gallery block, after inserting an image, the gallery block itself should be expected to receive focus, just like any other block. However, the loss of focus makes it difficult to seamlessly manipulate the block.

This problem occurs at least both when selecting images from the media library and when uploading media.

### Screenshots, screen recording, code snippet

https://user-images.githubusercontent.com/54422211/213845659-406a25f4-b7d2-4122-8281-9d9a7e920d7d.mp4

**Comment by ghost** [comment]
Is this a duplicate of [#27521](https://github.com/WordPress/gutenberg/issues/27521)? Cause in that case this one can be closed.

**Comment by t-hamano** [comment]
@tomdevisser
Thanks for pointing that out! I think this issue is exactly the same problem as #27521, so let's close it 👍
